packages/twenty-server/.env.test

```diff
@@ -21,3 +21,4 @@ REFRESH_TOKEN_SECRET=secret_refresh_token
 # MESSAGING_PROVIDER_GMAIL_ENABLED=false
 # STORAGE_TYPE=local
 # STORAGE_LOCAL_PATH=.local-storage
+# MAXIMUM_RECORD_AFFECTED=100
```

Comment:
Can we give a more specific name? Maybe `MUTATION_MAXIMUM_AFFECTED_RECORD` or something like that? Introducing mutation or graphql or pg_graphql in the name?

Reply:
You're right, it seems better !